website/versioned_docs/version-v3.7.x/externaldata.md

```diff
@@ -5,7 +5,7 @@ title: External Data
 
 > ❗ This feature is still in alpha stage, so the final form can still change (feedback is welcome!).
 
-> 🚧  Mutation is not yet supported with external data.
```

Comment:
Is this feature being backported to 3.7.x? If not, do we want to change that version's docs?

Reply:
Probably not backporting the feature. I can undo this change.